### ISSUE TITLE
Node stuck in a fork resolution

### ISSUE THREAD

**Issue by M5oul** [duniter/duniter#1104]
Using version 1.5.9, my node is stuck in a fork resolution at block #53481, although the HEAD's network is #53552:
```bash
2017-09-18T19:07:00+02:00 - info: ✘ PEER 4fHMTFBM
2017-09-18T19:07:00+02:00 - error: Unknown reference block of peer
2017-09-18T19:07:00+02:00 - error: Unknown reference block of peer
2017-09-18T19:07:00+02:00 - info: Blocks were not applied.
2017-09-18T19:07:00+02:00 - info: Blocks were not applied.
2017-09-18T19:07:00+02:00 - info: Blocks were not applied.
2017-09-18T19:07:00+02:00 - info: Blocks were not applied.
2017-09-18T19:07:00+02:00 - info: Block resolution: 1 potential blocks after current#53480...
2017-09-18T19:07:00+02:00 - error:  Error: ruleInputIsAvailable
    at Error (native)
    at Function.<anonymous> (/home/moul/projects/duniter/app/lib/blockchain/DuniterBlockchain.js:173:23)
    at next (native)
    at fulfilled (/home/moul/projects/duniter/app/lib/blockchain/DuniterBlockchain.js:4:58)
2017-09-18T19:07:00+02:00 - info: Fork resolution: 52 potential block(s) found...
2017-09-18T19:07:01+02:00 - info: Fork resolution: 3 potential suite(s) found...
2017-09-18T19:07:01+02:00 - info: Fork resolution: HEAD = block#53480
2017-09-18T19:07:01+02:00 - info: Fork resolution: suite 1/3 (-> #53530-000003) revert to fork point block#53480
2017-09-18T19:07:01+02:00 - info: Fork resolution: suite 1/3 REFUSED block#53481: ruleInputIsAvailable
2017-09-18T19:07:01+02:00 - info: Fork resolution: suite 2/3 (-> #53499-000003) revert to fork point block#53480
2017-09-18T19:07:01+02:00 - info: Fork resolution: suite 2/3 REFUSED block#53481: ruleInputIsAvailable
2017-09-18T19:07:01+02:00 - info: Fork resolution: suite 3/3 (-> #53495-000007) revert to fork point block#53480
2017-09-18T19:07:01+02:00 - info: Fork resolution: suite 3/3 REFUSED block#53481: ruleInputIsAvailable
2017-09-18T19:07:01+02:00 - info: Will pull blocks from the network in 0 min 20 sec
2017-09-18T19:07:02+02:00 - info: Matched 3 zeros 00027DD704EAEE9AE512AF25C32DF4D118BBAF51BC0B3252AD5B3C898558BE5F with Nonce = 10200000035759 for block#53481 by GfKERH
2017-09-18T19:07:09+02:00 - info: ⬇ PEER DJQjWdkU
2017-09-18T19:07:09+02:00 - info: ✘ PEER DJQjWdkU
2017-09-18T19:07:09+02:00 - error: Unknown reference block of peer
2017-09-18T19:07:09+02:00 - error: Unknown reference block of peer
2017-09-18T19:07:09+02:00 - info: ⬇ PEER DJQjWdkU
```

**Comment by M5oul** [comment]
Ah, too late. I trashed the data.
My node is now on the `WS2P` test with a new synchro.

**Comment by M5oul** [comment]
I close this one as I wasn't able to reproduce it.
I think it have been fixed in 1.6.0 release.
Please, reopen if you can reproduce it.
